expense-tracker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# set perms chmod u+x ./expense-tracker.sh
+# end immediately on non-zero exit codes
+set -e
+
+# setup local development
+if [[ $1 = 'up' ]]; then
+    sfdx force:org:create -a ScratchOrg -f config/project-scratch-def.json -s -d 28
+    sfdx force:source:push -f
+    sfdx force:user:create -f config/consultant-user-def.json
+    sfdx force:user:create -f config/finance-user-def.json
+    sfdx force:org:open
+
+elif [[ $1 = 'test' ]]; then
+    sfdx force:org:create -a TestOrg -f config/project-scratch-def.json
+    sfdx force:source:push -u TestOrg -f
+    sfdx force:user:create -f config/consultant-user-def.json
+    sfdx force:user:create -f config/finance-user-def.json
+    sfdx force:org:open -u TestOrg
```

Comment:
Nice job! I know this already merged, but please add a command to delete the test org at the end.

Reply:
@kikemoto - I was actually going to but then I was unsure because the commands all run consecutively, right? So it would delete the org right after standing it up and I (as the manual tester) wouldn't have time to open and check it, right? I can see if there is some sort of timeout parameter I should put on the delete though, would that be good?